<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceDialog</class>
 <widget class="QDialog" name="InvoiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invoice Generator</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="invoiceNumberLabel">
         <property name="text">
          <string>Invoice Number</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="invoiceNumberLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="projectLabel">
         <property name="text">
          <string>Project</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="amountLabel">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="amountLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="createDateLabel">
         <property name="text">
          <string>Create Date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDateEdit" name="createDateDateEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="paidLabel">
         <property name="text">
          <string>Paid</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="paidCheckBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="paymentDateLabel">
         <property name="text">
          <string>Payment Date</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDateEdit" name="paymentDateDateEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="notesLabel">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QTextEdit" name="notesTextEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="projectLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InvoiceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InvoiceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
